Monika After Story/game/sprite-chart.rpy

```diff
@@ -302,6 +302,10 @@ init -100 python in mas_sprites:
     # v: ignored
     # marks that a hair style has no tails. By default we assume ponytail.
 
+    EXP_H_NH = "no-hat"
+    #v: ignored
+    #marks the hair as not capable of having a hat
+
```

Comment:
This kind of prop is too broad. What characteristic of the braided hair makes it so marisa hat doesn't work with it? 

`no-hat` doesn't describe anything about the hairstyle itself, which makes it difficult for someone making a custom hairstyle to to determine if they need this prop or not.